static/src/javascripts/projects/common/modules/navigation/newHeaderNavigation.js

```diff
@@ -32,33 +62,24 @@ define([
         });
     }
 
-    function enhanceToButton() {
-        var checkboxes = qwery('.js-enhance-checkbox');
-        fastdom.read(function () {
-            var buttons = checkboxes.map(function (checkbox) {
-                var button = document.createElement('button');
-                var checkboxId = checkbox.id;
-                var checkboxControls = checkbox.getAttribute('aria-controls');
-                var checkboxClasses = Array.prototype.slice.call(checkbox.classList);
-
-                checkboxClasses.forEach(function (c) {
-                    button.classList.add(c);
-                });
-                button.setAttribute('id', checkboxId);
-                button.setAttribute('aria-controls', checkboxControls);
-                button.setAttribute('aria-expanded', 'false');
-
-                return button;
-            });
-            fastdom.write(function () {
-                buttons.forEach(function (button, index) {
-                    var checkbox = checkboxes[index];
-                    var eventHandler = buttonClickHandlers[button.id];
+    function enhanceCheckboxesToButtons() {
+        var toggles = [
```

Comment:
Nit (you don't have to listen to this): could we rename this variable `checkboxes`. As I am reading this code I keep forgetting what toggles are.